docs/source/events/payment-status-change.rst

```diff
@@ -6,6 +6,30 @@ This event means that the status of a payment has changed. To reduce noise, we o
 For each payment method, there are typical status sequences that the payment goes through. The exact meaning of the statuses and content of the ``payment_data`` also depends on the payment method.
 
 
+Line items
+----------
+
+.. versionadded:: 1.2.0
+
+Each status update contains the full list of payment ``line_items``. Each line item has the following data:
+
+- ``name``: A unique name for this line item
+- ``amount`` (float): The price of each unit
```

Comment:
Oh this is a float not an integer! The example higher up should be 10.0 then right?

Reply:
See the updated documentation of the float type.